index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>PraisyDram 2022</title>
+  <link
+    rel="stylesheet"
+    href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+  />
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <main>
+    <header class="header">
+      <i class="menu-icon fas fa-bars fa-2x"></i>
+      <a href="./index.html"><img src="./public/assets/icons/pd-logo.png" alt="PraisyDram Logo"></a>
+      <nav class="menu-items">
+        <i class="close fas fa-times"></i>
+        <ul>
+          <li><a href="./about.html" >About</a></li>
```

Comment:
- [ ] Please make sure your about menu option has the color same as on the design, It will guide the users through the project that the about page is active and users can visit it.